torch/_dynamo/optimizations/torchxla_integration.py

```diff
@@ -8,8 +8,7 @@
 
 import torch
 
-debug = os.environ.get("debug_extract_compiled_graph") == "1"
-
+debug = os.environ.get("debug_torchxla") == "1"
```

Comment:
maybe rename to `TORCH_XLA_DEBUG`?

although, i would also suggest you may not need a special variable.  just use `logging.DEBUG` and `logging.INFO`, then control `torch._dynamo.config.log_level` from torchdynamo side to affect what  output is shown here

Reply:
Got it. What I'm worried is that using the dynamo level log level may print too much things irrelevant to the debugging of the bridge.

Reply:
up to you